nginx-default.conf

```diff
@@ -2,7 +2,7 @@ server {
 	listen 80 default_server;
 	listen [::]:80 default_server;
 
-	root /var/www/html/mirror/public;
+	root /var/repos/mirror/public;
```

Comment:
If using the install prefix setting, it should modify the Nginx config file programmatically.

Please refer following steps:

- Revert the `nginx-default.conf` file.
- After running the [command](https://github.com/peter279k/mirror-kickstarter/blob/master/mirror-kickstarter.sh#L54), using the `sed` command to replace the `/var/www/html/mirror/public` setting with the `INSTALL_PREFIX` variable in the `/etc/nginx/sites-available/default`.